### ISSUE TITLE
Issue with grid 1-4

### ISSUE THREAD

**Issue by tartakxg** [IonDen/ion.rangeSlider#772]
`$("#example_id").ionRangeSlider(
{			type: "integer",
			min: 1,
			max: 4,
			from: 1,
			to: 4,
			postfix: "",
			hide_min_max: true,
			grid: true,
      }
);`

Grid has 1-2-3-3-4 and wrong steps:
https://jsfiddle.net/tartakx/a0rygw7x/

**Comment by IonDen** [comment]
Yes, in some cases automatic grid could have some problems. You could fix it by using `grid_snap: true` parameter.

**Comment by IonDen** [comment]
Released in 2.4.2, now on npm (https://www.npmjs.com/package/ion-rangeslider/v/2.4.2); cdnjs and jsDelivr pick it up from there. The full list of changes is in history.md (https://github.com/IonDen/ion.rangeSlider/blob/master/history.md). Thanks for the report.
